app/build.gradle

```diff
@@ -1,12 +1,13 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 32
+    namespace 'ai.elimu.appstore'
 
     defaultConfig {
         applicationId "ai.elimu.appstore"
-        minSdkVersion 24
-        targetSdkVersion 32
+        minSdkVersion 26
+        compileSdk 35
+        targetSdkVersion 35
```

Comment:
@jo-elimu By upgrading targetSdkVersion to 35, I assume you already verified that the behavior changes [here](https://developer.android.com/about/versions/15/behavior-changes-15) have been verified and updated (if any needed)?